app/src/features/admin/users/ActiveUsersList.tsx

```diff
@@ -305,7 +307,13 @@ const ActiveUsersList: React.FC<React.PropsWithChildren<IActiveUsersListProps>>
                 {activeUsers.length > 0 &&
                   activeUsers.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row, index) => (
                     <TableRow data-testid={`active-user-row-${index}`} key={row.id}>
-                      <TableCell>{row.user_identifier || 'No assigned role'}</TableCell>
+                      <TableCell>{row.user_identifier || 'No identifier'}</TableCell>
+                      <TableCell>
+                        <Typography sx={{ fontFamily: 'Monospace', display: 'inline' }}>
+                          {row.user_guid.substring(0, 4)}
+                        </Typography>
+                        {'\u0020\u0020\u2022\u2006\u2022\u2006\u2022\u2006'}
+                      </TableCell>
```

Comment:
Probably should run this by Jer, in case he has any strong opinions about how to display these

Also, we can probably show the full guids here if we want.
(I didnt run this, but I assume this is some kind of `1234 . . . .` credit card style display?)

Reply:
Removed.